src/wallet/feebumper.cpp

```diff
@@ -215,21 +215,19 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
     // We cannot source new unconfirmed inputs(bip125 rule 2)
     new_coin_control.m_min_depth = 1;
 
-    CTransactionRef tx_new;
-    CAmount fee_ret;
-    int change_pos_in_out = -1; // No requested location for change
     bilingual_str fail_reason;
     FeeCalculation fee_calc_out;
-    if (!wallet.CreateTransaction(recipients, tx_new, fee_ret, change_pos_in_out, fail_reason, new_coin_control, fee_calc_out, false)) {
+    auto txr = wallet.CreateTransaction(recipients, /* No requested location for change */ -1, fail_reason, new_coin_control, fee_calc_out, false);
```

Comment:
While it's arguably not, strictly speaking, part of the proposed change, the comment about the change position parameter that's being maintained here is a bit misleading. This is a bit nit-picky, so bare with me please, but I think it's more appropriate to:
1. Use a const instead of inline comment in order to convey the meaning. e.g. `const int RANDOM_CHANGE_POSITION = -1`). This value is used more than once, so another reason to use a const.
2. Convey that passing `-1` means "actively set the position to a random one" rather than "no requested location" which doesn't really let the reader know what's going to happen.

Reply:
Thanks a lot for reviewing, good point! I probably was a bit over-enthusiastic in getting rid of as many variables as possible before the call to `CreateTransaction` ;-) Introducing a `RANDOM_CHANGE_POSITION` constant makes a lot of sense to me, see latest force-push of the second commit. Not sure yet if putting the constant into the header is a good idea, as it is only used for one single method and hence many modules that don't need the constant would see it as well. Probably a candidate for a follow-up PR?